zebedee-cms/src/test/java/com/github/onsdigital/zebedee/model/publishing/PublisherTest.java

```diff
@@ -0,0 +1,50 @@
+package com.github.onsdigital.zebedee.model.publishing;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertTrue;
+
+public class PublisherTest {
+
+    private final String TEST_DIR_NAME = "src/test/resources/bpTestFolder";
+    private Publisher publisher;
+
+    @Before
+    public void setup() throws IOException {
+        this.publisher = new Publisher();
+    }
+
+    @Test
+    public void testConvertUriWithJsonForEvent() {
+
+        //Given {a single uri with data.json ready to be sent to kafka}
+        String testUri = "/testUri0/data.json";
+
+        //When {sending a uris to kafka}
+        String actuals = publisher.convertUriForEvent(testUri);
+
+        System.out.println(actuals);
+
+        //Then {uri does not have "data.json" string}
+        assertTrue(!actuals.contains("/testUri0/data.json"));
+        assertTrue(actuals.contains("/testUri0"));
```

Comment:
Use assertFalse rather than asserting `!` is true. But also is it needed anyway. You could just use assertEquals instead
```suggestion
        assertEquals("/testUri0",actuals);
```
NB. I'd rename `actuals` to `actual` too. 
PS. Same for other test